cpggen/cli.py

```diff
@@ -378,7 +378,7 @@ def cpg(
                 ret = []
                 exec_results = []
                 url = ""
-                # Where the source is a url or a CVE id download it
+                # Where the source is an url or a CVE id download it
```

Comment:
I think it is actually a not an because url is pronounced like 'you-r-l' not 'earl'